dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/UserDetails.java

```diff
@@ -171,8 +188,14 @@ default boolean isInUserDataHierarchy(String orgUnitPath) {
 
   private static boolean isInUserHierarchy(
       @CheckForNull String orgUnitPath, @Nonnull Set<String> orgUnitIds) {
-    if (orgUnitPath == null) return false;
-    for (String uid : orgUnitPath.split("/")) if (orgUnitIds.contains(uid)) return true;
+    if (orgUnitPath == null) {
+      return false;
+    }
+    for (String uid : orgUnitPath.split("/")) {
+      if (orgUnitIds.contains(uid)) {
+        return true;
+      }
+    }
```

Comment:
Do you have your IDE configured to insert brackets for all block statements?

Reply:
Sorry about that, it might be my IDE yeah. I will revert it.